<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>491</width>
     <height>691</height>
    </rect>
   </property>
   <property name="title">
    <string>Период Отчета</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>С:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditStart">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditEnd">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>По:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите период отчета:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>491</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string>               Выберите столбцы для отчета:</string>
   </property>
   <widget class="QPushButton" name="btnGenerate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать отчет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkName">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Товар</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkSupplier">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Поставщик</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkReceived">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Количество купленного</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkSold">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Количество проданного</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkStock">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Остаток</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkPrice">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Цена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkTotalPrice">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Потрачено</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkRevenue">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Выручка с продажи</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkDate">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Дата</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
